docs/src/content/docs/ja/components/asides.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: アサイド
```

Comment:
[0321c2e](https://github.com/withastro/starlight/pull/2384/commits/0321c2e3f1c55f30c79ebcd9f44dbe2a301891c4)

This was the part I was just struggling with when I implemented it...

I initially named it “アサイド” because I thought it would be better to treat it like a proper noun, but if you had documented it that way before, I think I'll use that name for now! sounds nice:)